### ISSUE TITLE
Support Request: no tablets were detected on NixOS

### ISSUE THREAD

**Issue by sb-child** [OpenTabletDriver/OpenTabletDriver#2305]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### What do you need help with?

OS: NixOS 22.05
`otd-gui` Version: 0.6.0.4
`otd-daemon` Version: 0.6.0.4

OpenTabletDriver can't detect `Gaomon 1060 Pro`
after trying to solve permission issues, i can't see any error but no tablets were detected:
![图片](https://user-images.githubusercontent.com/55868015/173194925-828363f2-ffcb-4671-a04a-43b3de5d10ed.png)
![图片](https://user-images.githubusercontent.com/55868015/173194934-7da198bc-267b-463f-a5f2-24bdc6970350.png)

`lsusb` output:
```
$ lsusb
Bus 003 Device 002: ID 256c:0064 Gaomon Gaomon Tablet_1060Pro
```

### What have you tried?

i tried to restart my computer, and add user-define udev on `configuration.nix`:
```
  services.udev = {
    # Gaomon 1060 Pro
    extraRules = ''
      SUBSYSTEM=="hidraw", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="006d", MODE="0666"
      SUBSYSTEM=="hidraw", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="0064", MODE="0666"
      SUBSYSTEM=="usb", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="006d", MODE="0666"
      SUBSYSTEM=="usb", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="0064", MODE="0666"
      SUBSYSTEM=="input", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="006d", ENV{LIBINPUT_IGNORE_DEVICE}="1"
      SUBSYSTEM=="input", ATTRS{idVendor}=="256c", ATTRS{idProduct}=="0064", ENV{LIBINPUT_IGNORE_DEVICE}="1"
    '';
  };
```

### Diagnostic Information

but it crashed:
```
(OpenTabletDriver.UX.Gtk:9375): GLib-GIO-ERROR **: 23:50:12.547: No GSettings schemas are installed on the system
[1]    9375 IOT instruction (core dumped)  otd-gui
```

**Comment by jamesbt365** [comment]
`otd getdiagnostics > diag.txt`

**Comment by sb-child** [comment]
> `otd getdiagnostics > diag.txt`
[diag.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8884278/diag.txt)

**Comment by jamesbt365** [comment]
otd getstring 9580 100 200

**Comment by sb-child** [comment]
> otd getstring 9580 100 200

![图片](https://user-images.githubusercontent.com/55868015/173195407-fc199e7e-93e9-4620-8faf-b83e01c28b81.png)
maybe need hexdump
![图片](https://user-images.githubusercontent.com/55868015/173195461-ede198a7-2e26-43d5-a906-f3c8bdefba2a.png)

**Comment by jamesbt365** [comment]
instead of 200 do 201, my mistake

**Comment by sb-child** [comment]
> 

```
$ otd getstring 9580 100 201
GM001_T213_210615
```

**Comment by jamesbt365** [comment]
This is indeed a different variant, let me make a configuration real quick.

**Comment by jamesbt365** [comment]
[1060 Pro.json.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8884302/1060.Pro.json.txt)
put this in `~/.local/share/OpenTabletDriver/Configurations`

**Comment by jamesbt365** [comment]
make sure to remove the .txt at the end

**Comment by sb-child** [comment]
> [1060 Pro.json.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8884302/1060.Pro.json.txt) put this in `~/.local/share/OpenTabletDriver/Configurations`

seems this file is not loaded. 
and i modified this file
![图片](https://user-images.githubusercontent.com/55868015/173195940-77bf83f2-cad1-4d74-9842-90bc66b6bb81.png)
but seems not found on [diag.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8884334/diag.txt)

maybe other way to load this configures?

**Comment by jamesbt365** [comment]
restart opentabletdriver

**Comment by sb-child** [comment]
> restart opentabletdriver

thank you. i found
```
ExecStart=/nix/store/4kv8y4di2pw705c6qijfq3zf2fq4a3ji-OpenTabletDriver-0.6.0.4/bin/otd-daemon -c /nix/store/4kv8y4di2pw705c6qijfq3zf2fq4a3ji-OpenTabletDriver-0.6.0.4/lib/OpenTabletDriver/Configurations
```
on `opentabletdriver.service`, so user-define config is not loaded.

i am running daemon manually and it is working
```
$ otd-daemon
...
[Device:Debug]  Initializing device 'Gaomon Tablet_1060Pro' /sys/devices/pci0000:00/0000:00:14.0/usb3/3-2/3-2:1.0/0003:256C:0064.0004/hidraw/hidraw1
[Device:Debug]  Using report parser type 'OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicReportParser'
[Device:Debug]  Initialized string index 200
[Detect:Info]   Found tablet 'Gaomon 1060 Pro update'
[Evdev:Debug]   Successfully initialized virtual tablet. (code NONE)
[Gaomon 1060 Pro update:Info]   Output mode: Absolute Mode
[Gaomon 1060 Pro update:Info]   Display area: [1920x1080@<960, 540>:0°],
[Gaomon 1060 Pro update:Info]   Tablet area: [254x158.75@<127, 79.375>:0°],
[Gaomon 1060 Pro update:Info]   Clipping: Enabled
[Gaomon 1060 Pro update:Info]   Ignoring reports outside area: Disabled
[Gaomon 1060 Pro update:Info]   Tip Binding: [Mouse Button Binding: Left]@0%
[Settings:Info] Driver is enabled.
```

**Comment by jamesbt365** [comment]
as the service is ran as the user and otd is told to check all locations it shouldn't have a problem with loading configs in the user directory.

Anyway fixed with #2239
